install_config/redeploying_certificates.adoc

```diff
@@ -391,6 +391,13 @@ $ ansible-playbook -i <inventory_file> \
     playbooks/openshift-etcd/redeploy-ca.yml
 ----
 
+[IMPORTANT]
+====
+After you run the `playbooks/openshift-etcd/redeploy-ca.yml` playbook for the first time, subsequent executions of the playbook do not renew the etcd CA. This is because the playbook does not overwrite the `/etc/etcd/etcd_ca.tgz` file, to prevent an accidental overwrite.
```

Comment:
@hexfusion instead of "to prevent an accidental overwrite" (which isn't helpful as it just repeats the earlier part of the sentence), do you have suggestions for better wording of what we're preventing by not overwriting that file?

Reply:
```
After you run the `playbooks/openshift-etcd/redeploy-ca.yml` playbook for the first time, a compressed bundle containing the CA signers will be persisted to `/etc/etcd/etcd_ca.tgz`. Because the CA signers are required for the generation of new etcd certificates it is very important that they are backed up.

If the playbook is ran again as a precaution it will not overwrite the bundle on disk. Please backup the bundle and move it from this path and the playbook will succeed.
```